tsconfig.json

```diff
@@ -3,27 +3,25 @@
     "target": "es6",
     "module": "commonjs",
     "moduleResolution": "node",
-    "isolatedModules": true,
-    "experimentalDecorators": true,
-    "emitDecoratorMetadata": true,
-    "declaration": false,
+    "isolatedModules": false,
     "noImplicitAny": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
     "removeComments": false,
-    "noLib": true,
     "preserveConstEnums": true,
     "suppressImplicitAnyIndexErrors": true,
-    "outDir": "lib"
+    "lib": ["es2017"],
+    "outDir": "./lib",
+    "declaration": true,
+    "sourceMap": true,
+    "pretty": true
   },
-  "filesGlob": [
-    "src/**/*.ts",
-    "!node_modules/**",
-    "custom_typings/*.d.ts",
-    "typings/main/*.d.ts",
-    "typings/main.d.ts",
-    "node_modules/typescript/lib/lib.core.es6.d.ts"
+  "include": [
+    "src/**/*.ts"
   ],
-  "files": [],
-  "atom": {
-    "rewriteTsconfig": true
-  }
+  "exclude": [
```

Comment:
This isn't strictly speaking necessary, as it's not in the include section. IMO, drop.